bluebrain/repo-patches/packages/neuron/package.py

```diff
@@ -42,14 +42,13 @@ class Neuron(CMakePackage):
 
     variant("binary",     default=True, description="Create special as a binary instead of shell script (8.0.x and earlier)")
     conflicts("~binary", when='@8.0.999:')
-    variant("coreneuron", default=False, description="Enable CoreNEURON support")
+    variant("coreneuron", default=True, description="Enable CoreNEURON support")
```

Comment:
This is one extra change as it alters the hh.mod file but I think this is ok and a long pending change anyway.

Reply:
I have a branch in `NEURON` and `CoreNEURON` that fixes this issue with the enablement of `TABLE` in the various cases. I think it still needed some work and that's why I didn't create a PR but I will do in a free Friday (after the repo merge). Shouldn't be a lot of work pending

Reply:
Ok, we can discuss it. But I guess enabling coreneuron by default here is fine, right?

Reply:
Yes. In our stack everything runs with `CoreNEURON` enabled so I don't expect to see any floating point differences with that enabled